cypress/e2e/createEditTenure.feature

```diff
@@ -1,8 +1,11 @@
+@tenure
 @authentication
 @common
-@processes
 @root
-@tenure
+@search
+@worktray
```

Comment:
Could you give link to this MFE?
Would want to click around it. I'm quite curious as to whether this is actually connected to `createEditTenure`.